xbmc/settings/AdvancedSettings.h

```diff
@@ -154,7 +154,8 @@ class CAdvancedSettings : public ISettingCallback, public ISettingsHandler
     float m_videoIgnorePercentAtEnd;
     float m_audioApplyDrc;
     unsigned int m_maxPassthroughOffSyncDuration = 10; // when 10 ms off adjust
-    bool m_AllowMultiChannelFloat = false; // Android only switch to be remved in v22
+    bool m_AllowMultiChannelFloat = false; // Android only switch to be removed in v22
+    bool m_superviseAudioDelay = false; // Android only to not correct broken audio firmwares
```

Comment:
```suggestion
    bool m_superviseAudioDelay = false; // Android only to correct broken audio firmwares
```

or maybe:
// Android only workaround to correct broken audio firmwares

since both code to supervise delay and setting to disable it are workarounds